tf_sender:
  ros__parameters:
    pair_to_ip:
      "['map','cpsl_uav_1/odom']": "100.110.146.134"
      #"['map','cpsl_robot_dog_1/odom']": "100.110.146.134"
    pair_to_replace:
      "['map','cpsl_uav_1/odom']":
        frame_id: "map"
        child_frame_id: "cpsl_uav_1/odom"
      #"['map','cpsl_robot_dog_1/odom']":
      #  frame_id: "map"
      #  child_frame_id: "odom"

waypoint_sender:
  ros__parameters:
    robot_id_to_ip:
      cpsl_uav_1: "100.110.146.134"
      #cpsl_robot_dog_1: "100.110.146.134"

map_sender:
  ros__parameters:
    target_ips:
      - "100.110.146.134"
      #- "100.110.146.134"
